flexget/plugins/filter/abort_exists.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+import re
+
+from flexget import plugin
+from flexget.event import event
+
+log = logging.getLogger('abort_exists')
+
+class PluginAbortExists(object):
+    """Aborts a task if an entry field matches the regexp"""
+
+    schema = {
+        'type': 'object',
+        'properties' : {
+            'regexp': {'type': 'string', 'format': 'regex'},
+            'field': {'type': 'string'}
+        },
+        'required': ['regexp', 'field'],
+        'additionalProperties': False
+    }
+
+    # Execute as the first thing in filter phase
+    @plugin.priority(255)
+    def on_task_filter(self, task, config):
+        abort_re = re.compile(config['regexp'], re.IGNORECASE)
+        field = config['field']
+        for entry in task.all_entries:
+            # if pattern matches any entry
+            if field not in entry:
+                log.debug('Field %s not found. Skipping.', field)
+            if abort_re.search(entry[field]):
+                task.abort('An entry contained %s in field %s. Abort!' % (config['regexp'], field))
```

Comment:
Dunno. I kind of want logging but I also don't want to add a separate log entry.